images/build_images.sh

```diff
@@ -242,13 +242,31 @@ configure_gcp_project() {
   log "Packer will be using service account ${CLOUD_BUILD_ACCOUNT}"
 
   # Add cloudbuild SA permissions
-  gcloud projects add-iam-policy-binding "${GCP_PROJECT}" \
-    --member "${CLOUD_BUILD_ACCOUNT}" \
-    --role roles/compute.instanceAdmin.v1
+  CHECK_FIRST_PERMISSION=$(
```

Comment:
Could we name this variable CHECK_ROLE_INST_ADMIN and the other one CHECK_ROLE_SVC_ACCT so that the variable describes the role that is being checked?

Reply:
Thanks for the suggestion. Variable names updated.